lib/ansible/modules/system/mount.py

```diff
@@ -1,20 +1,18 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 
-# (c) 2012, Red Hat, inc
+# Copyright: (c) 2012, Red Hat, inc
 # Written by Seth Vidal
 # based on the mount modules from salt and puppet
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-
 ANSIBLE_METADATA = {'metadata_version': '1.1',
```

Comment:
Please format this like this:

```python
ANSIBLE_METADATA = {
    'metadata_version': '1.1',
    'status': ['preview'],
￼    'supported_by': 'core'
}
```

Reply:
This block is not maintained by me. Every module has the block defined like this.

Reply:
It has this format because this is what `pprint` produces. But the [documentation](http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#version-1-1-of-the-metadata) is showing different formatting.

Reply:
Well, you are free to change it. But that part of the file was added by the core team as-is. And almost all modules have it like that.